cmd/buildah/commit.go

```diff
@@ -20,11 +20,20 @@ var (
 			Usage: "don't compress layers",
 		},
 		cli.StringFlag{
+			Name:  "cert-dir",
+			Value: "",
+			Usage: "use certificates at the specified path to access the registry",
+		},
+		cli.StringFlag{
 			Name:  "creds",
 			Value: "",
 			Usage: "use `username[:password]` for accessing the registry",
 		},
 		cli.StringFlag{
```

Comment:
Shouldn't this be a bool? And it needs to default to true, according to man page below.

Reply:
I think you're right.  WIll go play with that.

Reply:
Fixed, good catch ty.  Changed to BoolTFlag which defaults to true.